wowless/sandbox.c

```diff
@@ -0,0 +1,88 @@
+#include <lauxlib.h>
+
+/* TODO proper elune header inclusion */
+extern void luaL_openlibsx(lua_State *, int);
```

Comment:
One change I'm considering (and actually did - but then backed out of a little while ago) is to move the extension APIs to their own headers, namely:

* `lelune.h` for core extensions (any function with a "lua_" prefix)
* `leluneaux.h` for auxilliary library functions (any function with "luaL_" prefix)

Would this make things any easier?

Reply:
No. This TODO really is a plea to sanity for include paths when building C modules. The fix here is in the rockspec (or complete abandonment thereof), not elune.